src/Compilers/CSharp/Portable/Symbols/Source/SourceEventAccessorSymbol.cs

```diff
@@ -246,11 +248,5 @@ protected string GetOverriddenAccessorName(SourceEventSymbol @event, bool isAdde
 
             return null;
         }
-
-        internal override bool IsExpressionBodied
-        {
-            // Events cannot be expression-bodied
-            get { return false; }
-        }
```

Comment:
this is just wrong (but was overridden, so it wasn't a problem).  Event accessors can def be expression-bodied.